modules/python/test/test_misc.py

```diff
@@ -326,6 +326,20 @@ def test_parse_to_cstring_not_convertible(self):
             with self.assertRaises((TypeError), msg=get_no_exception_msg(not_convertible)):
                 _ = cv.utils.dumpCString(not_convertible)
 
+    def test_parse_to_string_convertible(self):
+        try_to_convert = partial(self._try_to_convert, cv.utils.dumpString)
+        for convertible in (None, '', 's', 'str', str(123), np.str('test1'), np.str_('test2')):
```

Comment:
> None

do we really need this?

Reply:
All existed conversion functions are (even for trivial types such as `int`, `float` and etc.) very tolerant to `None` values to Python. In this test I wanted to explicitly notice this in test, because if this behaviour will be changed sometimes - it will be clear for API users - tests will document this.

Reply:
"tested" usually means that feature is supported without future regressions.
"non-tested" - undocumented behavior, it may work for now, use it at own risk.

`None` handling is still not complete in Python binding implementation. See [this comment](https://github.com/opencv/opencv/pull/15915#discussion_r353205637) from previous PR.
So this `None` case should go away after implementation got fixed.